example/geometry/t8_example_geometries.cxx

```diff
@@ -118,6 +118,14 @@ class t8_geometry_sincos: public t8_geometry {
     SC_ABORT_NOT_REACHED ();
   }
 
+  void
```

Comment:
Why not put this routine as virtual function into `t8_geometry`. Then all derived geometry classes have this routine and do not have to put this everywhere.